presentation-pt-br.Rmd

```diff
@@ -74,11 +74,11 @@ Algumas boas razões para patrocinar um satRday:
 
 + O satRday é sem fins lucrativos: patrocínios e lucros (se houver) serão __investidos em eventos da comunidade__. 
 
-+ O satRday é "baixo custo": queremos __estabelecer a inscrição pelo menor preço possível__. Isso só é possível devido aos nossos patrocinadores. 
++ O satRday é acessível: queremos __que a inscrição tenha o menor preço possível__. Isso só é possível devido aos nossos patrocinadores. 
 
 + O satRday está "na vanguarda": esta conferência tem como objetivo convidar palestrantes que são __líderes em seu campo de especialização__. 
```

Comment:
eu ainda não gosto muito dessa `vanguarda` mas n sei o que colocar no lugar

Reply:
Também achei esquisita essa palavra.. mas tbm não sei o que colocar no lugar

Reply:
em inglês: "at the cutting edge", caso alguém tenha alguma sugestão melhor